manticore/core/plugin.py

```diff
@@ -32,7 +32,7 @@ class Tracer(Plugin):
     def will_start_run_callback(self, state):
         state.context['trace'] = []
 
-    def did_execute_instruction_callback(self, state, pc, target_pc, instruction):
+    def will_execute_instruction_callback(self, state, pc, instruction):
```

Comment:
Does this works on forking instructions ? Does this get executed twice on a fork ?

Reply:
yes it probably will. i don't have time to really think too much about a proper solution for this right now, and it's not that high priority, so i'll omit this diff

Reply:
good catch, thanks!